src/lib/logical_query_plan/show_tables_node.cpp

```diff
@@ -12,6 +16,15 @@ std::shared_ptr<AbstractLQPNode> ShowTablesNode::_on_shallow_copy(LQPNodeMapping
   return ShowTablesNode::make();
 }
 
+const std::vector<std::shared_ptr<AbstractExpression>>& ShowTablesNode::column_expressions() const {
```

Comment:
?

Reply:
Ran into #1137 as well, fixed it and didn't mention it ;)